api/src/main/java/io/hyperfoil/function/ToStringFunction.java

```diff
@@ -0,0 +1,13 @@
+package io.hyperfoil.function;
+
+@FunctionalInterface
+public interface ToStringFunction<T> {
+
+    /**
+     * Applies this function to the given argument.
+     *
+     * @param value the function argument
+     * @return the function result
+     */
+    String applyAsString(T value);
```

Comment:
You don't need a special `ToStringFunction`, a regular `Function<String, T>` is fine.

Reply:
Removed `ToStringFunction `